dag_universities_f.py

```diff
@@ -22,16 +22,19 @@
 # Conexion a la base de datos
 engine = create_engine(f'postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}')
 
-'''
+def extract():
+    '''
     Ejecuta una consulta SQL y almacena
     el resultado en la carpeta files con formato csv
-'''
-def extract():
+    '''
     route = os.path.dirname(__file__)
     file_sql = f'{route}/sql/universidades-f.sql'
     with open(file_sql, 'r') as query:
         sql_query = query.read()
-    df = pd.read_sql_query(sql_query, engine)
+    df = pd.read_sql_query(sql_query, engine)        
+    # Pregunto si la carpeta existe y sino la crea 
+    if not(os.path.exists(f"{route}/files")):
+        os.makedirs(f"{route}/files")        
```

Comment:
Esta bien la validación, pero se podría hacer con la propia funcionalidad makedirs, pasándole el parámetro exist_ok=True. te dejo mas detalle:

Syntax: os.makedirs(path, mode = 0o777, exist_ok = False) 
Parameter: 
path: A path-like object representing a file system path. A path-like object is either a string or bytes object representing a path.
mode (optional) : A Integer value representing mode of the newly created directory..If this parameter is omitted then the default value Oo777 is used.
exist_ok (optional) : A default value False is used for this parameter. If the target directory already exists an OSError is raised if its value is False otherwise not. For value True leaves directory unaltered. 
Return Type: This method does not return any value.